app/Http/Requests/ItemImportRequest.php

```diff
@@ -42,7 +42,18 @@ public function import(Import $import)
         $import->field_map  = request('column-mappings');
         $import->save();
         $fieldMappings=[];
+
         if ($import->field_map) {
+            foreach ($import->field_map as $field => $fieldValue) {
+                $errorMessage = null;
+
+                if(is_null($fieldValue)){
+                    $errorMessage = "The Import Field for: " . $field . ", shouldn't be empty";
```

Comment:
Can you turn this into a translation string instead?

Also what happens if the user wants to import a null value here?

Reply:
The value is not affected, this validation is for the fields that the user is trying to import, that never can be null, as in the screenshot attached.

<img width="924" alt="Captura de pantalla 2019-03-10 a la(s) 0 25 23" src="https://user-images.githubusercontent.com/653557/54303418-8d4ccc00-4588-11e9-8879-5e277572b13d.png">